plugins/database/mysql/mysql_test.go

```diff
@@ -483,35 +718,6 @@ func TestMySQL_UpdateUser(t *testing.T) {
 	}
 }
 
-func TestMySQL_Initialize_ReservedChars(t *testing.T) {
```

Comment:
This test was added for [coverage against a bug](https://github.com/hashicorp/vault/pull/8040) that we ended up fixing and is not specific to templating, but rather a check against [this codeblock](https://github.com/hashicorp/vault/blob/master/sdk/database/helper/connutil/sql.go#L66-L69). We might want to keep it around.

Reply:
This is being checked in the `TestMySQL_Initialize` table tests as the password being used is the same as in this tests. I'll see about making this check more explicit though. I'd like to keep it in the table tests, but TBD on whether that's reasonable.

Reply:
Looks like you've addressed this on 716ddc4, thanks!